ZenPacks/zenoss/WSMAN/datasources/WSMANDataSource.py

```diff
@@ -262,12 +261,12 @@ def onSuccess(self, results, config):
 
         data['events'].append({
             'eventClassKey': 'wsmanCollectionSuccess',
-            'eventClass': '/Status/Events',
+            'eventClass': ds0.eventClass or '/Status/Events',
```

Comment:
Do we need this part ‘or “/Status/Events”’?

Reply:
You would want to set the default eventClass on WSMAN datasource level, an example -https://github.com/zenoss/ZenPacks.zenoss.CalculatedPerformance/blob/develop/ZenPacks/zenoss/CalculatedPerformance/datasources/AggregatingDataSource.py#L37